sql/parse/parse.go

```diff
@@ -2484,45 +2484,61 @@ func tableExprToTable(
 		return expression.NewUnresolvedTableFunction(t.Name, exprs), nil
 
 	case *sqlparser.JoinTableExpr:
-		// TODO: add support for using, once we have proper table
-		// qualification of fields
-		if len(t.Condition.Using) > 0 {
-			return nil, sql.ErrUnsupportedFeature.New("USING clause on join")
-		}
+		return handleJoinTableExpr(ctx, t)
 
-		left, err := tableExprToTable(ctx, t.LeftExpr)
-		if err != nil {
-			return nil, err
+	case *sqlparser.ParenTableExpr:
+		if len(t.Exprs) == 1 {
```

Comment:
if we only ever have one, does `*ast.ParenTableExpr.Exprs` need to be a list? or can it be `.Expr`?

Reply:
it's possible to have multiple table expressions in `ParenTableExpr` because it references to list of `table_reference`, but i think for join, it can only be one table_reference.